_posts/2023-06-26-trino-fest-2023-salesforce.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Trino Fest 2023 recap: Anomaly detection for Salesforce’s production data using Trino"
+author: "Tuli Nivas, Geeta Shankar, and Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Salesforce.png
+show_pagenav: false
+---
+
+Rolling into our next presentation of [Trino Fest 2023]({% post_url
+2023-06-20-trino-fest-2023-recap %}), we're excited to bring you
+Tuli Navas and Geeta Shankar's talk from the Performance Engineering Team at
+Salesforce. They provide numerous reasons for why they need Trino and
+further explain how it is essential for anomaly detection in
+their data. It's an insightful talk about using a query engine to ensure data
+quality and how switching to Trino has massively improved their performance.
+You definitely don't want to miss it.
+
+<!--more-->
+
+{% youtube nFuqpb2GjVI %}
+
+## Recap
+
+Salesforce provides customer relationship management software and applications
+focused on sales, customer service, marketing automation, e-commerce, analytics,
+and application development. They host hundreds of thousands of customers that
+generate millions of transactions per day. For a company of this size, they
+need a query engine that is fast and efficient. During the talk, Tuli made it
+clear how much Salesforce relies on Trino, stating, "Trino has been a one-stop
+shop for analytics." Trino was the perfect solution for them, as it allows them
```

Comment:
There was a mention in this talk about Trino running gnarly queries or so.. that might be a good quote to add.